vertical-selection.kak

```diff
@@ -62,7 +62,7 @@ define-command -hidden vertical-selection-impl -params 2 %{
         # extend (resp. reverse extend) to all lines that match the pattern (or are short enough)
         # or stop if the next (resp. previous) line is not a candidate
         reg / "(?S)(?:%reg{p}|%arg{2})"
-        exec "%arg{1}<ret><a-x>"
+        exec "%arg{1}<ret>"
```

Comment:
This change doesn't seem to work in new `kak`, e.g. for selecting leading spaces on a line and extending down. Following is the 1-1 change for new `kak` but I didn't test how it behaves with the previous versions:
```suggestion
        exec "%arg{1}<ret>x"
```

Reply:
Yeah in this case I'm just going to give up on backwards compatibility and document the version requirement